### ISSUE TITLE
Tabs not described in VoiceOver

### ISSUE THREAD

**Issue by sandymcfadden** [wordpress-mobile/WordPress-iOS#5414]
This appears to be very similar to https://github.com/wordpress-mobile/WordPress-iOS/issues/5078
When scrolling through the tabs with voice over they are described as only "Tab 1 of 5" or "Tab 2 of 5 selected". There is no indication what each tab actually does.

**Comment by rachelmcr** [comment]
When I test in WPiOS 6.1, I do get accurate tab descriptions like this (while on My Sites tab):

- "Selected My Sites tab 1 of 5"
- "Reader tab 2 of 5"
- "New post tab 3 of 5"
- "Me tab 4 of 5"
- "Notifications tab 5 of 5"

However, this is broken in the latest beta 6.2.0.20160511. I get these tab descriptions instead:

- "Selected tab 1 of 5"
- "Tab 2 of 5"
- "Tab 3 of 5"
- "Tab 4 of 5"
- "Notifications tab 5 of 5" (correct)

#####Tested on iPhone 6, iOS 9.3.1, WPiOS 6.1 and 6.2.0.20160511

cc @astralbodies

**Comment by astralbodies** [comment]
@sendhil I think we need to consider not releasing 6.2 if this is the case.

**Comment by astralbodies** [comment]
It looks like my fix doesn't work on iOS 9.3.2. @jleandroperez confirmed 6.1 and 6.2 both work on 9.3.1.

**Comment by astralbodies** [comment]
@rachelmcr @sandymcfadden Can you try following this procedure:

1. Turn off VO.
1. Force quit the app.
1. Turn on VO.
1. Restart the app.
1. Verify tab labels are being spoken.

I think its a glitch in 9.3.2 with turning on VO when the app is already launched.

**Comment by rachelmcr** [comment]
I confirmed those steps resolve the issue for me — the tab labels are now being spoken.

I then tested these steps:

1. Force quit the app.
2. Turn off VO.
3. Restart the app.
4. Turn on VO.

Result: Tab labels aren't being spoken.

So your diagnosis sounds correct based on the behavior I see.

**Comment by astralbodies** [comment]
I'm going to leave this issue open until I report it to Apple in a Radar.

**Comment by astralbodies** [comment]
Closing this issue as I'm just going to bring back the visual labels.
